app/views/Home.scala.html

```diff
@@ -37,7 +37,9 @@
 	home_tile: home_tile
 )
 
-@(nextPaymentDueDate: Option[LocalDate],
+@(
```

Comment:
I am fine with this approach, but would be nice to move all these parameters into a single view model